getAmazon_title_price.py

```diff
@@ -0,0 +1,19 @@
+import bs4, requests
+
+def getAmazonPrice(productUrl): # Take in the URL of the Amazon page
+    headers = {
+    'User-Agent': 'My User Agent 1.0',
+    'From': 'youremail@domain.com'  # This is another valid field
```

Comment:
Try removing this. It might not be needed.

Reply:
@plumenator I found that without passing the header with a user-agent that the scraper gets blocked by Amazon's captcha; in some cases I even needed to pass an accept-language